src/client/OpenFin/apps/Launcher/NlpSuggestions/executions/RfqExecution/RfqExecution.tsx

```diff
@@ -9,7 +9,8 @@ import { onNext, useRfqExecutionState } from "./state"
 
 const Usage = () => (
   <HelpText>
-    Usage: <Pill>buy/sell</Pill> <Pill>quantity</Pill> <Pill>bond</Pill>
+    Usage: <Pill>buy/sell</Pill> <Pill>quantity</Pill> <Pill>bond</Pill>{" "}
+    <Pill>maturity</Pill>
```

Comment:
we discussed making this optional .. I don't see any code for that
and we should show that it is optional here .. and need to state that it is 2-digit year, don't we? or will it work with 2 or 4?

Reply:
You make it optional in the dialogflow (so wont be seen in the code) and yes ill add in the helper text that it is an optional parameter (2-digit)